app/handler/handler.go

```diff
@@ -137,6 +131,23 @@ func buildAndSaveMessage(msg *textsecure.Message, syncMessage bool) {
 	store.UpdateSession(session)
 	m.ExpireTimer = msg.ExpireTimer()
 	m.HTime = helpers.HumanizeTimestamp(m.SentAt)
+	if msg.Quote() != nil {
+		msgFlags = helpers.MsgFlagQuote
+		text = msg.Message()
+		err, id := store.FindQuotedMessage(msg.Quote())
+		if err != nil || id == -1 {
+			// create quoted message
+			quoteMessage := session.Add(text, msg.Quote().GetAuthorE164(), nil, msg.Quote().GetText(), false, store.ActiveSessionID)
+			quoteMessage.Flags = helpers.MsgFlagHiddenQuote
+			err, savedQuoteMessage := store.SaveMessage(quoteMessage)
+			log.Debugln("id ", id)
```

Comment:
Maybe be a bit more verbose? "Quoted message id" or something like that? Also, you're changing the id the line below, so should this log be before, or after that?

Reply:
Best is to not logging anything, because we it's not necessary. So i removed it